client/src/main/scala/com/aliyun/emr/rss/client/write/LifecycleManager.scala

```diff
@@ -1359,7 +1359,7 @@ class LifecycleManager(appId: String, val conf: RssConf) extends RpcEndpoint wit
 
   def checkAlive(): Boolean = {
     try {
-      rssHARetryClient.askSync[CheckAliveResponse](CheckAlive, classOf[CheckAliveResponse]).alive
+      rssHARetryClient.askSync[CheckAliveResponse](CheckAlive(), classOf[CheckAliveResponse]).alive
```

Comment:
I think reference to a case object should not add braces, and I've check spark :)